rdrf/rdrf/services/rest/serializers.py

```diff
@@ -54,6 +58,12 @@ class Meta:
         }
         exclude = ('rdrf_registry', 'working_groups', 'created_by')
 
+    def get_clinical_data(self, instance):
+        entry = ClinicalData.objects\
+            .filter(django_model="Patient", django_id=instance.id, collection="cdes")\
+            .order_by("last_updated_at").first()
+        return entry.data if entry else {}
```

Comment:
Probably we'd need to add some post-processing here to make the data easier to understand for the users of the API. I'm thinking of some CDE values which have a PermitterValueGroup and in the clinical data the permitted value code is stored instead of the actual value. Similart things for some CDEs with custom widgets. For example when  using the DurationWidget the actual value is in ISO 8601 duration format.

Reply:
https://eresearchqut.atlassian.net/browse/ERE-909